tests/flytekit/unit/extras/tasks/test_shell.py

```diff
@@ -249,3 +252,81 @@ def test_shell_script():
 
     assert t.script_file == script_sh
     t(f=test_csv, y=testdata, j=datetime.datetime(2021, 11, 10, 12, 15, 0))
+
+
+def test_raw_shell_task_with_args(capfd):
+    if script_sh_2 is None:
+        return
+    pst = get_raw_shell_task()
```

Comment:
can you write an example of the usage in a workflow, for better understanding and maybe, we should add example later to [Flytesnacks](https://github.com/flyteorg/flytesnacks/blob/master/cookbook/core/flyte_basics/shell_task.py)  to evnagelize this

Reply:
Sure, where should I save the example? Should I open a PR into flytesnacks with the example?